doc/ref_arch/kubernetes/chapters/chapter02.rst

```diff
@@ -1435,7 +1434,7 @@ Kubernetes Architecture Requirements
 
 The requirements in this section are to be delivered in addition to those in section `Reference Model
 Requirements <#reference-model-requirements>`_, and have been
-created to support the Principles defined in the :ref:`chapters/chapter01:Overview` of this
+created to support the Principles defined in the :ref:`first chapter <chapters/chapter01:Overview>` of this
```

Comment:
```suggestion
created to support the Principles defined in the :ref:`first chapter <chapters/chapter01:Overview>` chapter of this
```

Reply:
Would that not then read 'first chapter chapter' when rendered?

Reply:
I am in agreement with @tomkivlin